quickstarts/ocs-install-tour-quickstart.yaml

```diff
@@ -0,0 +1,58 @@
+---
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: ocs-install-tour
+spec:
+  version: 4.7
+  displayName: Installing Openshift Container Storage
+  durationMinutes: 5
+  icon: data:image/svg+xml;base64,PHN2ZyBlbmFibGUtYmFja2dyb3VuZD0ibmV3IDAgMCAxMDAgMTAwIiBoZWlnaHQ9IjEwMCIgdmlld0JveD0iMCAwIDEwMCAxMDAiIHdpZHRoPSIxMDAiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+PHBhdGggZD0ibTY2LjcgNTUuOGM2LjYgMCAxNi4xLTEuNCAxNi4xLTkuMiAwLS42IDAtMS4yLS4yLTEuOGwtMy45LTE3Yy0uOS0zLjctMS43LTUuNC04LjMtOC43LTUuMS0yLjYtMTYuMi02LjktMTkuNS02LjktMy4xIDAtNCA0LTcuNiA0LTMuNSAwLTYuMS0yLjktOS40LTIuOS0zLjIgMC01LjIgMi4xLTYuOCA2LjYgMCAwLTQuNCAxMi41LTUgMTQuMy0uMS4zLS4xLjctLjEgMSAuMSA0LjcgMTkuMiAyMC42IDQ0LjcgMjAuNm0xNy4xLTZjLjkgNC4zLjkgNC44LjkgNS4zIDAgNy40LTguMyAxMS40LTE5LjEgMTEuNC0yNC42IDAtNDYuMS0xNC40LTQ2LjEtMjMuOSAwLTEuMy4zLTIuNi44LTMuOS04LjkuNS0yMC4zIDIuMS0yMC4zIDEyLjIgMCAxNi41IDM5LjIgMzYuOSA3MC4yIDM2LjkgMjMuOCAwIDI5LjgtMTAuNyAyOS44LTE5LjIgMC02LjctNS44LTE0LjMtMTYuMi0xOC44IiBmaWxsPSIjZWQxYzI0Ii8+PHBhdGggZD0ibTgzLjggNDkuOGMuOSA0LjMuOSA0LjguOSA1LjMgMCA3LjQtOC4zIDExLjQtMTkuMSAxMS40LTI0LjYgMC00Ni4xLTE0LjQtNDYuMS0yMy45IDAtMS4zLjMtMi42LjgtMy45bDEuOS00LjhjLS4xLjMtLjEuNy0uMSAxIDAgNC44IDE5LjEgMjAuNyA0NC43IDIwLjcgNi42IDAgMTYuMS0xLjQgMTYuMS05LjIgMC0uNiAwLTEuMi0uMi0xLjh6IiBmaWxsPSIjMDEwMTAxIi8+PC9zdmc+
+  description: Install the OCS operator and create a storage cluster to have the OCS service up and running.
+  introduction: Red Hat OpenShift Container Storage is persistent software-defined storage integrated with and optimized for Red Hat OpenShift Container Platform. Dynamic, stateful, and highly available container-native storage can be provisioned and de-provisioned on demand as an integral part of the OpenShift administrator console.
+  tasks:
+  - title: Installing the OpenShift Container Storage
+    description: |- 
+      The OperatorHub is where you can find a catalog of available Operators to install on your cluster.
+      
+      Follow these steps to Install OpenShift Container Storage
+
+      **To expand the OCS storage cluster follow these steps:**
+      
+      1. Visit the OperatorHub and search for the Openshift container storage operator.
+      2. Click on the operator option to initiate the operator installation on your cluster.
+      3. Click install when you're finished with Operator Subscription form.
+      4. When installation is finished click **Create Storage Cluster**.
+    review:
+      instructions: |-
+        ####  Verify that the OpenShift Container Storage operator is installed. 
+
+        Does the status of the installed operator show as Succeeded?
+      failedTaskHelp: This task isn’t verified yet. Try the task again.
+    summary:
+      success: You have installed the Openshift Container Storage operator!`
```

Comment:
Is this a stray backtick at the end of the sentence?

Reply:
removed